nodestats.go

```diff
@@ -75,15 +76,16 @@ func (ds *eventDispatchStats) CombinedStats(bufferStats map[t.ModuleID]int) []in
 // ==============================================================================================================
 
 // monitorStats prints and resets the dispatching statistics every given time interval, until the node is stopped.
-func (n *Node) monitorStats(interval time.Duration) {
+func (n *Node) monitorStats(interval time.Duration, wg *sync.WaitGroup) {
+	defer wg.Done()
+
 	ticker := time.NewTicker(interval)
 
-Loop:
 	for {
 		select {
 		case <-n.workErrNotifier.ExitC():
 			ticker.Stop()
```

Comment:
just wondering, could we not print the stats here too? Not awfully useful, but could perhaps help with debugging once the node fails.

Reply:
Smart! Done.